packages/invoices/src/OutboundShipment/DetailView/SidePanel/SidePanel.tsx

```diff
@@ -14,7 +14,7 @@ import { RelatedDocumentsSection } from './RelatedDocumentsSection';
 export const SidePanelComponent = () => {
   const { success } = useNotification();
   const t = useTranslation('distribution');
-  const { data } = useOutbound();
+  const { data } = useOutbound.document.get();
```

Comment:
that's very neat - I like how clear that is! the `.get()` to return data makes sense